op-chain-ops/genesis/layer_two_test.go

```diff
@@ -73,6 +73,10 @@ func testBuildL2Genesis(t *testing.T, config *genesis.DeployConfig) *core.Genesi
 		require.Equal(t, common.Big1, gen.Alloc[addr].Balance)
 	}
 
```

Comment:
If [these](https://github.com/ethereum-optimism/op-geth/blob/ed8e9f54ed0a5069e77ea34895ff12a9ac4ca17f/consensus/misc/create2deployer.go#L19) were public, then we would be able to use constants to make assertions here

Reply:
Happy to refactor. I feel like these constants should be part of the superchain repo.